src/executors/helm-chart-testing.yaml

```diff
@@ -0,0 +1,3 @@
+---
+docker:
+  - image: quay.io/helmpack/chart-testing:v2.4.0
```

Comment:
How about renaming this fine `chelmpack-chart-testing.yaml` to have something more structured?. We also need to retag this image.

Reply:
I'll retag and rename